README.md

```diff
@@ -62,7 +62,7 @@
 
 ## 🚀 Live Demo <a name="live-demo"></a>
 
-- Project isn't deployed yet but you can check how to [install it](#getting-started).
+- [https://jorgeabrahan.github.io/microverse_portfolio/](https://jorgeabrahan.github.io/microverse_portfolio/)
```

Comment:
- [ ] Nicely done, your deployed link works fine, and it is present in the readme, congratulations @jorgeabrahan, if I may, I have some suggestions to propose to you in case you will be interested to follow them, but I would strongly recommend you do so ❤️, 
- [ ]  - [ OPTIONAL] It is about the readme itself, I think since we are at the end of the HTML and CSS milestone, it would be fine to make a few improvements to your readme file to look a little bit more awesome and attractive, I will drop a link for you to take an example from, please find it [here](https://github.com/joskalenda/crisp-coorporate-company), YOU CAN MAKE IT EVEN BETTER